all-the-icons.el

```diff
@@ -224,6 +224,7 @@
 
     ("\\.p[ml]$"        all-the-icons-alltheicon "perl"                 :face all-the-icons-lorange)
     ("\\.pl6$"          all-the-icons-fileicon "perl6"                  :face all-the-icons-cyan)
+    ("\\.pm6$"          all-the-icons-fileicon "perl6"                  :face 'all-the-icons-pink)
```

Comment:
Can you remove the single quote on the all the lines please? They are not needed?

Reply:
Of course. Give me a couple of minutes.

Reply:
Done! let me know if you need anything else